satellite/console/service.go

```diff
@@ -41,11 +44,14 @@ type Service struct {
 	store DB
 	log   *zap.Logger
 
+	mail *mailservice.Service
+
+	templatePath string
 	passwordCost int
 }
 
 // NewService returns new instance of Service
-func NewService(log *zap.Logger, signer Signer, store DB, passwordCost int) (*Service, error) {
+func NewService(log *zap.Logger, signer Signer, store DB, mail *mailservice.Service, templatePath string, passwordCost int) (*Service, error) {
```

Comment:
`console` probably shouldn't care about how templates are handled, keep that information inside mailservice.